cocos/2d/components/label.ts

```diff
@@ -439,8 +439,12 @@ export class Label extends Renderable2D {
         // if delete the font, we should change isSystemFontUsed to true
         this._isSystemFontUsed = !value;
```

Comment:
Why do these properties need to change together
Each property should take care of itself

Reply:
Syntactically, _userDefinedFont is just a field instead of an attribute, it doesn't have getter or setter.
Functionally, _userDefinedFont is a auxiliary variable for usesystemfont and font attributes, its assignments should be synchronized with the assignments of font.